pkg/leader/leader.go

```diff
@@ -5,6 +5,7 @@ import (
 	"os"
 	"time"
 
+	"github.com/pkg/errors"
```

Comment:
We do not want to use the package [github.com/pkg/errors](https://github.com/pkg/errors). It was made obsolete by go 1.13 when they added error wrapping to the standard lib. And the package has since been archived. Instead, we can use `fmt.Errorf("failed: %w", err)`

Reply:
Fixed